ArduPlane/quadplane.cpp

```diff
@@ -424,6 +424,7 @@ static const struct AP_Param::defaults_table_struct defaults_table[] = {
     { "Q_A_RAT_PIT_FILT", 10.0 },
     { "Q_M_SPOOL_TIME",   0.25 },
     { "Q_M_HOVER_LEARN",     0 },
+    { "Q_M_THST_EXPO",     0.2 },
```

Comment:
I think this param is the expo used for to got from thrust to throttle for each motor, so we shouldn't use it for this also. I think copter does have a parameter for what your using it for tho, maybe we need a new param for Quadplanes

http://ardupilot.org/copter/docs/motor-thrust-scaling.html

Reply:
I can't find a equivalent param for copter so probably it would be fine to just hard code it to what ever copter uses

Reply:
you're right; I'll create a new param for Plane, since I didn't like the hard-coded value in copter.